apps/blocklist/models.py

```diff
@@ -152,6 +152,22 @@ def flush_urls(self):
         return ['/blocklist*']  # no lang/app
 
 
+class BlocklistIssuerCert(BlocklistBase, amo.models.ModelBase):
+    _type = 'c'
+    issuer = models.TextField()  # annoyingly, we can't know the size
```

Comment:
We try to make comments as sentences (start with an uppercase, end with a dot):
`# Annoyingly, we can't know the size.`